common/static/common/js/discussion/views/discussion_thread_list_view.js

```diff
@@ -570,6 +569,20 @@
                                 gettext('Show posts by {username}.'), {username: username}
                             );
                             self.addSearchAlert(message, 'search-by-user');
+                            DiscussionUtil.safeAjax({
+                                url: DiscussionUtil.urlFor('followed_threads', response.users[0].id),
```

Comment:
@noraiz-anwar Firstly I thought this. if we add this code `searchForUser` without changing the order of execution it makes three ajax request, In my point of view, it's not good. In this case, its only make two ajax request.